kubernetes/.swagger-codegen-ignore

```diff
@@ -1,8 +1,8 @@
 .gitignore
 git_push.sh
-# Remove this once swagger-codegen 2.2.3 is released and we update. 
+# Remove this once swagger-codegen 2.3.0 is released and we update. 
```

Comment:
The change is not merged yet, which is why those files are in that ignore list. Once 2.3.0 is out (or the PR is merged) it will be safe to regenerate the files.